hollow/src/main/java/com/netflix/hollow/core/write/objectmapper/HollowListTypeMapper.java

```diff
@@ -68,6 +69,7 @@ public int write(Object obj) {
         if(ignoreListOrdering) {
             IntList ordinalList = getIntList();
             for(Object o : l) {
+                Objects.requireNonNull(o, "Null element. Lists cannot contain null elements");
```

Comment:
Nice. Is it possible to add the list type name to the message? Similar comments for the map below - might help track down which list/map has null elements or keys/values